src/ConEmuHk/ShellProcessor.cpp

```diff
@@ -2266,7 +2266,7 @@ int CShellProc::PrepareExecuteParms(
 	else
 	{
 		// хотят GUI прицепить к новой вкладке в ConEmu, или новую консоль из GUI
-		if (lbGuiApp && (NewConsoleFlags || bForceNewConsole))
+		if ((bLongConsoleOutput) || (lbGuiApp && (NewConsoleFlags || bForceNewConsole)))
```

Comment:
This is absolutely wrong. The condition check for starting [ChildGui](https://conemu.github.io/en/ChildGui.html) in ConEmu tab, don't change it!